chromium_src/components/search_engines/template_url_prepopulate_data.cc

```diff
@@ -81,6 +81,13 @@ const std::vector<BravePrepopulatedEngineID> brave_engines_AU_IE = {
     PREPOPULATED_ENGINE_ID_ECOSIA,
 };
 
+const std::vector<BravePrepopulatedEngineID> brave_engines_KR = {
```

Comment:
I wonder why these are vectors instead of plain arrays?

Reply:
Ok, I'll try to handle this as a f/u. Filed https://github.com/brave/brave-browser/issues/27943